specification/containerregistry/resource-manager/readme.md

```diff
@@ -26,14 +26,23 @@ These are the global settings for the ContainerRegistry API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2023-08-preview
+tag: package-preview-2023-11
 ```
 
+
+### Tag: package-preview-2023-11
+
+These settings apply only when `--tag=package-preview-2023-11` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-11'
+input-file:
+  - Microsoft.ContainerRegistry/preview/2023-11-01-preview/containerregistry.json
+```
 ### Tag: package-2023-08-preview-only
```

Comment:
@raymondluo1 we are missing the package with the Tasks swagger. See the example for 23-08 release. There is a package `package-2023-08-preview` that has just the ACR Core + ACR tasks swagger. Then there is `package-2023-08-preview-only` that just has the ACR swagger.  Please do the same for 23-11-preview.